salt/kubernetes-master/init.sls

```diff
@@ -100,14 +100,19 @@ kube-controller-manager:
     - source:      salt://kubernetes-master/addons/namespace.yaml.jinja
     - template:    jinja
 
-/root/skydns-rc.yaml:
+/root/kubedns-cm.yaml:
```

Comment:
Again, I know this is all existing code, however, can we avoid putting stuff in /root/? /etc/kubernetes/caasp-addons or something could be used, or maybe /var/lib/kubernetes/caasp-addons ?